cfgov/v1/models/learn_page.py

```diff
@@ -122,7 +122,7 @@ class Meta:
 
 class EventPage(AbstractFilterPage):
     # General content fields
-    body = RichTextField(blank=True)
+    body = RichTextField('Sub-Heading',blank=True)
```

Comment:
I really think renaming the field is better than changing the label. Either way still requires a migration and we can still remain consistent. _stopping now_